docs/docs-content/integrations/longhorn.md

```diff
@@ -17,10 +17,74 @@ Longhorn creates a dedicated storage controller for each block device volume and
 ## Version Supported
 
 <Tabs queryString="versions">
+<TabItem label="1.5.x" value="1.5.x">
+
+
+## Prerequisites
+
+- Kubernetes cluster is 1.21 or higher.
+
+## Parameters
+
+The table lists commonly used parameters you can configure when adding this pack.
+
+| Parameter                | Description                                            | Default                                     |
+|-------------------------|--------------------------------------------------------|---------------------------------------------|
+| `namespace`               | The namespace where the Longhorn will be installed. | `longhorn-system` |
+| `defaultClass`            | The volume type to be used. | `true` |
+| `defaultFsType`           | The default file system.    | `ext4` |
+| `defaultClassReplicaCount`| The default number of copies of data store in your cluster.   | `3`         |
+| `defaultDataLocality`     | The default location where data computation will occur. | `disabled` Best effort |
```

Comment:
This is a little confusing to me. If it's a location, why is the default "disabled"? And maybe we can expand on the "best effort" a bit?

Reply:
Agree, I'm gonna also link to their parameter page.